client/wfnews-war/src/main/angular/src/app/components/wf-stats-component/wf-stats.component.ts

```diff
@@ -25,8 +27,32 @@ export class WFStatsComponent implements OnInit {
 
   private updateSubscription: Subscription;
 
+  public FIRE_CENTRE_COLOURS = [
+    { name: 'Coastal', value: '#519a97' },
+    { name: 'Northwest', value: '#a27ea8' },
+    { name: 'Prince George', value: '#799136' },
+    { name: 'Kamloops', value: '#e96b56' },
+    { name: 'Southeast', value: '#369dc9' },
+    { name: 'Cariboo', value: '#ca8321' }
+  ]
+
+  public CAUSE_COLOURS = [
+    { name: 'Lightning', value: '#519a97' },
+    { name: 'Person', value: '#a27ea8' },
+    { name: 'Unknown', value: '#799136' }
+  ]
+
+  public STAGE_OF_CONTROL_COLOURS = [
+    { name: 'Fire of Note', value: '#519a97' },
+    { name: 'Out of Control', value: '#a27ea8' },
+    { name: 'Being Held', value: '#799136' },
+    { name: 'Under Control', value: '#e96b56' },
+    { name: 'Out', value: '#369dc9' }
+  ]
```

Comment:
Probably would be good idea to store these colors In _variables.scss if we will reuse them ?